code/modules/surgery/organs/wound.dm

```diff
@@ -11,41 +11,20 @@
 
 	// amount of damage this wound causes
 	var/damage = 0
-	// ticks of bleeding left.
```

Comment:
This is still WIP; I need to migrate damage to be handled entirely by the limbs themselves and get rid of limb stumps.

Once that is done, and tested, this *won't* be here.